### ISSUE TITLE
Not able to switch between sub-menus at the same level

### ISSUE THREAD

**Issue by rossib94** [FrDH/mmenu-js#1028]
Hi There,

Thanks for helping with my issue last week. I have discovered another issue that I am struggling to solve.

I currently have a 3 level menu, the problem is if I open a level 3 sub-menu and then click on another menu item with a level 3 sub-menu it won't close the first and show the 2nd. Is there a way to achieve this?

An example on my staging site is if you go to Store and open Paramotors and then try and open Trikes, it won't open.

My staging URL is:: [http://parajetstaging.wpengine.com/](url) you will need the following details to access that staging website::
User: demo
Password: c5138d74dac4

The code before mmenu does its changes is::
```html
<ul id="main-menu" class="">
   <li id="menu-item-26" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-26"><a href="http://localhost/parajet/?page_id=7"><img src="http://localhost/parajet/wp-content/uploads/New-Pilot-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>New Pilot</span></a></li>
   <li id="menu-item-27" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-27"><a href="http://localhost/parajet/?page_id=8"><img src="http://localhost/parajet/wp-content/uploads/Existing-Pilot-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>Existing Pilot</span></a></li>
   <li id="menu-item-28" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-28"><a href="http://localhost/parajet/?page_id=9"><img src="http://localhost/parajet/wp-content/uploads/Instructor-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>Instructor</span></a></li>
   <li id="menu-item-29" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-29"><a href="http://localhost/parajet/?page_id=10"><img src="http://localhost/parajet/wp-content/uploads/Adventure-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>Adventure</span></a></li>
   <li id="menu-item-30" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current_page_ancestor menu-item-has-children current_page_parent menu-item-30">
      <a href="http://localhost/parajet/?page_id=11"><img src="http://localhost/parajet/wp-content/uploads/Store-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>Store</span></a>
      <ul class="sub-menu">
         <li id="menu-item-32" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat current-product-ancestor current-menu-ancestor current-menu-parent current-product-parent menu-item-has-children menu-item-32">
            <a href="http://localhost/parajet/?product_cat=paramotors">Paramotors</a>
            <ul class="sub-menu">
               <li id="menu-item-73" class="menu-item menu-item-type-post_type menu-item-object-product current-menu-item menu-item-73"><a href="http://localhost/parajet/?product=maverick" aria-current="page">Maverick</a></li>
               <li id="menu-item-74" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-74"><a href="http://localhost/parajet/?product=zenith">Zenith</a></li>
               <li id="menu-item-75" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-75"><a href="http://localhost/parajet/?product=volution-3">Volution 3</a></li>
            </ul>
         </li>
         <li id="menu-item-33" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-has-children menu-item-33">
            <a href="http://localhost/parajet/?product_cat=trikes">Trikes</a>
            <ul class="sub-menu">
               <li id="menu-item-76" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-76"><a href="http://localhost/parajet/?product=falco">Falco</a></li>
            </ul>
         </li>
         <li id="menu-item-34" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-34"><a href="http://localhost/parajet/?product_cat=propellers">Propellers</a></li>
         <li id="menu-item-35" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-has-children menu-item-35">
            <a href="http://localhost/parajet/?product_cat=parajet-spares">Parajet Spares</a>
            <ul class="sub-menu">
               <li id="menu-item-77" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-77"><a href="http://localhost/parajet/?product_cat=maverick">Maverick</a></li>
               <li id="menu-item-78" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-78"><a href="http://localhost/parajet/?product_cat=zenith">Zenith</a></li>
               <li id="menu-item-79" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-79"><a href="http://localhost/parajet/?product_cat=volution-3">Volution 3</a></li>
               <li id="menu-item-80" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-80"><a href="http://localhost/parajet/?product_cat=falco">Falco</a></li>
               <li id="menu-item-81" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-81"><a href="http://localhost/parajet/?product_cat=throttles">Throttles</a></li>
               <li id="menu-item-82" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-82"><a href="http://localhost/parajet/?product_cat=harnesses">Harnesses</a></li>
            </ul>
         </li>
         <li id="menu-item-36" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-has-children menu-item-36">
            <a href="http://localhost/parajet/?product_cat=vittorazi-spares">Vittorazi Spares</a>
            <ul class="sub-menu">
               <li id="menu-item-83" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-83"><a href="http://localhost/parajet/?product=atom-80">Atom 80</a></li>
               <li id="menu-item-84" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-84"><a href="http://localhost/parajet/?product=moster-185-classic">Moster 185 Classic</a></li>
               <li id="menu-item-85" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-85"><a href="http://localhost/parajet/?product=moster-185-plus">Moster 185 Plus</a></li>
               <li id="menu-item-86" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-86"><a href="http://localhost/parajet/?product=moster-185-plus-my19">Moster 185 Plus MY’19</a></li>
               <li id="menu-item-87" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-87"><a href="http://localhost/parajet/?product=moster-185-factory">Moster 185 Factory</a></li>
               <li id="menu-item-88" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-88"><a href="http://localhost/parajet/?product=moster-185-factory-my19">Moster 185 Factory MY’19</a></li>
               <li id="menu-item-89" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-89"><a href="http://localhost/parajet/?product=moster-185-silent">Moster 185 Silent</a></li>
               <li id="menu-item-90" class="menu-item menu-item-type-post_type menu-item-object-product menu-item-90"><a href="http://localhost/parajet/?product=moster-185-silent-my19">Moster 185 Silent MY’19</a></li>
            </ul>
         </li>
         <li id="menu-item-37" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-37"><a href="http://localhost/parajet/?product_cat=wings">Wings</a></li>
         <li id="menu-item-38" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-38"><a href="http://localhost/parajet/?product_cat=reserve-chutes">Reserve Chutes</a></li>
         <li id="menu-item-39" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-39"><a href="http://localhost/parajet/?product_cat=bags-covers">Bags &amp; Covers</a></li>
         <li id="menu-item-40" class="menu-item menu-item-type-taxonomy menu-item-object-product_cat menu-item-40"><a href="http://localhost/parajet/?product_cat=clothing">Clothing</a></li>
         <li id="menu-item-41" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-41"><a href="http://localhost/parajet/?page_id=20">FAQ</a></li>
         <li id="menu-item-44" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-44"><a href="http://localhost/parajet/?page_id=23">Shipping</a></li>
         <li id="menu-item-43" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-43"><a href="http://localhost/parajet/?page_id=24">Returns</a></li>
         <li id="menu-item-42" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-42"><a href="http://localhost/parajet/?page_id=25">Payment</a></li>
      </ul>
   </li>
   <li id="menu-item-31" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-31"><a href="http://localhost/parajet/?page_id=12"><img src="http://localhost/parajet/wp-content/uploads/Props-Icon-1.svg" class="_mi _before _svg" aria-hidden="true"><span>About Us</span></a></li>
</ul>
```

**Comment by FrDH** [comment]
I get an 500 internal server error when visiting
http://parajetstaging.wpengine.com/product-category/paramotors/

**Comment by rossib94** [comment]
Fixed the internal server error but for me clicking the Paramotors link just opens level 3 of the menu, i'm using Chrome. I feel the way I have achieved that is hacky... I have made the .mm-btn_next sit over the top of the <a> that has the page URL. Is there a better way? I had a feeling that other browsers would struggle.

**Comment by FrDH** [comment]
still getting an internal server error :/

**Comment by FrDH** [comment]
Ok now I see the issue, I'll have to investigate as I don't see any reason for that not to work. 
If you want the entire menu item to open up the submenu, you can replace the `<a>` with a `<span>`. If you can't (I dont think WP lets you do that easily), your approach should work just as well.

**Comment by FrDH** [comment]
Found the bug, thank you for reporting it, will be fixed in the next version.

**Comment by rossib94** [comment]
Thank you very much for looking into this issue. Have you got an estimate for when the next release will be?

**Comment by FrDH** [comment]
should be fixed in 8.2
